config/routes.rb

```diff
@@ -9,4 +11,6 @@
   get    '/cart',                     to: 'carts#index'
   resources :products
   resources :users
+  resources :account_activations, only: [:edit]
+  resources :password_resets,     only: [:new, :create, :edit, :update]
 end
```

Comment:
Try to indent a little bit here. Some empty lines would help. For example, you can put an empty line before the `resources`

Reply:
What do you mean by indent? You recommend having the 'only's on the same line offset as the 'to's?